### PR TITLE
fix: 修复图片发送问题

### PR DESCRIPTION
**Pull request by leochen-g**

### CODE REVIEW

File: src/puppet-oa.ts [hunk 1]
```diff
@@ -480,8 +480,13 @@ class PuppetOA extends PUPPET.Puppet {
     file     : FileBox,
   ): Promise<string> {
     let msgtype: OAMediaType
-    switch (file.mimeType) {
+    const mimeType = file.mediaType
+    switch (mimeType) {
       case 'image/jpeg':
+      case 'image/jpg':
+      case 'image/png':
+      case 'image/webp':
```
Comment (willin) [inline src/puppet-oa.ts:488]: ? how to

**Comment by huan** [comment]
Please make sure the following requirements have been met for your PR before we can continue reviewing it:

1. You have signed the CLA by clicking the `[CLA | not signed yet]` button from the comments.
2. The PR has passed all the CI tests and turns _green_. (you can execute all tests locally by running `npm test`)

File: src/puppet-oa.ts (hunk 1 above)
Comment (leochen-g) [reply src/puppet-oa.ts:488]: 除了 webp 测试不太行之外，其他测试都已经通过了，其实最终都是转换成 jpg 格式了
image
image

**Comment by leochen-g** [comment]
> Please make sure the following requirements have been met for your PR before we can continue reviewing it:
> 
> 1. You have signed the CLA by clicking the `[CLA | not signed yet]` button from the comments.
> 2. The PR has passed all the CI tests and turns _green_. (you can execute all tests locally by running `npm test`)

因为测试用例使用的是 @huan 的测试号，但是现在测试号是不在线的，所以目前测试案例跑不通 😂

**Review by huan — APPROVED** [review]
LGTM

thanks for the improvements!

**Pull request merged**